packages/app-desktop/gui/NoteEditor/commands/showLocalSearch.ts

```diff
@@ -13,7 +13,6 @@ export const runtime = (comp: any): CommandRuntime => {
 				comp.editorRef.current.execCommand({ name: 'search' });
 			} else {
 				comp.setShowLocalSearch(true);
-				if (comp.noteSearchBarRef.current) comp.noteSearchBarRef.current.wrappedInstance.focus();
```

Comment:
Now that you made focus() public, does it work if you just replace this with `comp.noteSearchBarRef.current.focus()`?

Reply:
@laurent22 no it doesn't. We can't focus immediately after calling `comp.setShowLocalSearch(true)` because React needs some time to rerender, so I used the `useEffect` instead and called `noteSearchBarRef.current.focus()` in the `useEffect`.